src/BenchmarkRecord.py

```diff
@@ -177,3 +177,34 @@ def copy(self) -> any:
         :rtype: BenchmarkRecord
         """
         return deepcopy(self)
+
+
+class BenchmarkRecordStored:
+    """
+    This class can be used to store the benchmark record of a previous QUARK run as read from results.json.
```

Comment:
This class can be used to store the BenchmarkRecord of a previous QUARK run as read from results.json.
It is a simple wrapper with the purpose to provide the same interface to the BenchmarkManager as the
BenchmarkRecord does.